db/seeds.rb

```diff
@@ -1,7 +1,11 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Sosie.destroy_all
+User.destroy_all
+
+user = User.new({email: "contact@sark.com", name: "Sarkozy", biography: Faker::Marketing.buzzwords})
+user.password = "123456"
+user.save!
+10.times do
+  Sosie.create!({address: Faker::Address.full_address, name: Faker::Name.name, description: Faker::Marketing.buzzwords, price_per_day: Faker::Number.number(digits: 4), user: User.first})
+end
+
+puts "done"
```

Comment:
Can you add an empty line at the end of your code, please?

Reply:
Yes ofc